src/core/agent/agent.types.ts

```diff
@@ -181,6 +181,7 @@ enum NotificationRoute {
   MultiSigRpy = "/multisig/rpy",
   ExnIpexApply = "/exn/ipex/apply",
   ExnIpexAgree = "/exn/ipex/agree",
+  IpexAdmit = "/ipex/admit",
```

Comment:
Ah, actually the notification route would be `/exn/ipex/admit` and the exchange route is `/ipex/admit`.

In case this causes bugs, can we have a separate enum:

```
ExchangeRoute {
  IpexAdmit = "/ipex/admit"
}
```

exn and notification are different but use similar routes